services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.js

```diff
@@ -109,8 +109,11 @@ export const ExplosivesPermitForm = (props) => {
   );
 
   const isHistoric = !props.initialValues?.explosives_permit_id && props.isPermitTab;
-  const isAdmin = props.userRoles.includes(USER_ROLES[Permission.ADMIN]);
-  const disabled = props.isProcessed && !isAdmin;
+  const isESUP = props.userRoles.includes(USER_ROLES[Permission.EDIT_EXPLOSIVES_PERMITS]);
+  // eslint-disable-next-line no-unused-vars
+  const hasEditPermission = isESUP;
+  // TODO: BE fix required before enabling the form for anyone to edit
+  const disabled = props.isProcessed; // props.isProcessed && !hasEditPermission;
```

Comment:
Also for the future: disable editing on FE since BE will fail. And switch to use the disable variable to disable the form.